.gitignore

```diff
@@ -2,3 +2,4 @@
 *.log
 env/
 venv/
+*.iml
```

Comment:
Ah. That's an artifact of me editing the stuff with IntelliJ, which creates an `.iml` file to keep its project specific settings. Included it out of habit, but it can be removed, of course.